packages/dds/merge-tree/src/attributionPolicy.ts

```diff
@@ -190,13 +190,17 @@ function createPropertyTrackingMergeTreeCallbacks(
 
 function combineMergeTreeCallbacks(callbacks: AttributionCallbacks[]): AttributionCallbacks {
 	return {
-		delta: (...args) => callbacks.forEach(({ delta }) => delta(...args)),
-		maintenance: (...args) => callbacks.forEach(({ maintenance }) => maintenance(...args)),
+		delta: (...args): void => {
+			for (const { delta } of callbacks) delta(...args);
+		},
+		maintenance: (...args): void => {
+			for (const { maintenance } of callbacks) maintenance(...args);
+		},
 	};
 }
 
 /**
- * @returns An {@link AttributionPolicy} which tracks only insertion of content.
+ * Creates an {@link AttributionPolicy} for inserted content only.
```

Comment:
Current version makes it sound a bit like the policy applies to inserted content only, when in reality it means the only thing that is tracked is the initial insertion.
```suggestion
 * Creates an {@link AttributionPolicy} which only tracks initial insertion of content.
```